src/Application/Console/Formatters/Console.php

```diff
@@ -45,34 +46,56 @@ final class Console implements Formatter
     private const MIN_SPACEWIDTH = 5;
     private const MAX_SPACEWIDTH = 15;
 
-    /**
-     * @var Style
-     */
-    private $style;
-    /**
-     * @var int
-     */
-    private $totalWidth;
-    /**
-     * @var FileLinkFormatter
-     */
-    private $fileLinkFormatter;
-    /**
-     * @var bool
-     */
-    private $supportHyperLinks;
-    /**
-     * @var \NunoMaduro\PhpInsights\Domain\Configuration
-     */
-    private $config;
+    private const SUBTITLE = 'fg=white;options=bold;fg=white';
+
+    private const QUALITY = <<<EOD
+    <%quality_color%>%block_size%</>
+    <fg=black;options=bold;%quality_color%>  %quality%  </>
+    <%quality_color%>%block_size%</>
+    EOD;
+
+    private const COMPLEXITY = <<<EOD
+    <%complexity_color%>%block_size%</>
+    <fg=black;options=bold;%complexity_color%>  %complexity%  </>
+    <%complexity_color%>%block_size%</>
+    EOD;
+
+    private const STRUCTURE = <<<EOD
+    <%structure_color%>%block_size%</>
+    <fg=black;options=bold;%structure_color%>  %structure%  </>
+    <%structure_color%>%block_size%</>
+    EOD;
+
+    private const STYLE = <<<EOD
+    <%style_color%>%block_size%</>
+    <fg=black;options=bold;%style_color%>  %style%  </>
+    <%style_color%>%block_size%</>
+    EOD;
+
+    private const CATEGORY_COLOR = [
+        'Code' => 'cyan',
+        'Complexity' => 'green',
+        'Architecture' => 'blue',
+        'Style' => 'yellow',
+        'Security' => 'red',
+    ];
```

Comment:
Love these constants :D

Reply:
Yeah, actually really helpful that they are now defined up here instead 👍